config/app.config

```diff
@@ -65,7 +65,7 @@ ehcache.peerDiscovery = "PubSub"
 redis.host = "redis-srv"
 ehcache.replicate = true
 // point a IRSA for testing - I will be overriddien in a real deployment
-lsst.hips.masterUrl="https://irsa.ipac.caltech.edu/data/hips/list"
+lsst.hips.masterUrl="https://data-int.lsst.cloud/api/hips/list"
```

Comment:
I would use data.lsst.cloud here and then, when needed, override it in the Kubernetes config for data-int.

Reply:
ok, but it should be always overridden.

Reply:
In the actual config files, there's a template variable for "instance root URL" used, so in practice there's no need for pretty much ever typing "data-int" directly in the config files.

Reply:
- name: "PROPS_lsst__hips__masterUrl"
   value: "{{ .Values.global.baseUrl }}/api/hips/list"

https://github.com/lsst-sqre/phalanx/blob/13da8890c38bc3f3289e55102fda6649b85edfa9/services/portal/templates/deployment.yaml#L49

Reply:
See below - I get what you're doing now.